Source/KNSoft.SlimDetours/Memory.c

```diff
@@ -116,12 +117,21 @@ detour_memory_init(VOID)
         {
 #if defined(_WIN64)
             /* 1GB after Ntdll.dll */
-            PLDR_DATA_TABLE_ENTRY NtdllLdrEntry;
+            PVOID NtdllBase = NULL;
+            LdrGetDllHandle(NULL, NULL, (PUNICODE_STRING)&g_usNtdllDll, &NtdllBase);

```

Comment:
```suggestion
            LdrGetDllHandleEx(LDR_GET_DLL_HANDLE_EX_UNCHANGED_REFCOUNT,
                              NULL,
                              NULL,
                              (PUNICODE_STRING)&g_usNtdllDll,
                              &NtdllBase);
```
Use `LDR_GET_DLL_HANDLE_EX_UNCHANGED_REFCOUNT` flag to keep the original behavior (keep the refcount unchanged).

Reply:
AFAIK `LdrGetDllHandle(...)` is equivalent to `LdrGetDllHandleEx(LDR_GET_DLL_HANDLE_EX_UNCHANGED_REFCOUNT, ...)`.

In any case, I adapted your `GetNtdllBase` function. Regarding performance, it's only called once upon initialization so it shouldn't matter in practice, but I don't mind.